recipes/level-zero/all/conanfile.py

```diff
@@ -0,0 +1,64 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.files import apply_conandata_patches, get, export_conandata_patches
+
+
+class LevelZeroConan(ConanFile):
+    name = "level-zero"
+    license = "MIT"
+    homepage = "https://github.com/oneapi-src/level-zero"
```

Comment:
if this can only be built as a shared library (judging by https://github.com/oneapi-src/level-zero/blob/master/source/CMakeLists.txt#L14-L19) - then I think the package type needs to be set to `shared-library`